docs/reference/api/containers/splitcontainer.rst

```diff
@@ -20,30 +22,49 @@ Usage
 
     import toga
 
-    split = toga.SplitContainer()
     left_container = toga.Box()
     right_container = toga.ScrollContainer()
 
-    split.content = [left_container, right_container]
+    split = toga.SplitContainer(content=[left_container, right_container])
 
-Setting split direction
------------------------
-
-Split direction is set on instantiation using the ``direction`` keyword argument. Direction is vertical by default.
+Content can be specified when creating the widget, or after creation by assigning
+the ``content`` attribute. The direction of the split can also be configured, either
+at time of creation, or by setting the ``direction`` attribute:
 
 .. code-block:: python
 
     import toga
 
     split = toga.SplitContainer(direction=toga.SplitContainer.HORIZONTAL)
```

Comment:
Should this be changed to `Direction.HORIZONTAL`, if the `SplitContainer` attributes are only there for backward compatiblity?